settings.gradle.kts

```diff
@@ -1,21 +1,4 @@
-/*
- * Copyright (c) 2023 Contributors to the Eclipse Foundation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
```

Comment:
Why SPDX and header removed, will it make oru SPDX check explode later?

Reply:
Good eye, was automatically removed by the cherry pick since the ticket was already done in the old repo. Changing it.